sematic/config/server_settings.py

```diff
@@ -43,6 +44,10 @@ class ServerSettingsVar(AbstractPluginSettingsVar):
     # uses for jobs.
     SEMATIC_WORKER_KUBERNETES_SA = "SEMATIC_WORKER_KUBERNETES_SA"
 
+    # What, if any, imagePullSecrets should be used for runner and
+    # standalone jobs?
+    WORKER_IMAGE_PULL_SECRETS = "WORKER_IMAGE_PULL_SECRETS"
```

Comment:
Pleas use this opportunity to refresh the public-facing settings documentation such as https://docs.sematic.dev/diving-deeper/cli#server-settings
It's outdated, not only having incomplete lists of settings, but also mentions `server.yaml` / `user.yaml`, which we moved away from.